config/routes.rb

```diff
@@ -53,6 +53,8 @@
       end
     end
     get 'profile' => 'users#profile', :as => 'profile'
+    get 'jobs' => 'jobs#index', as: :jobs
```

Comment:
lets remove this old ruby syntax

Reply:
why? i could make it into a resource- but why? this is not a long term solution